pytorch_lightning/strategies/ddp.py

```diff
@@ -87,6 +89,7 @@ def __init__(
         ddp_comm_wrapper: Optional[callable] = None,
         model_averaging_period: Optional[int] = None,
         process_group_backend: Optional[str] = None,
+        timeout: Optional[timedelta] = default_pg_timeout,
```

Comment:
Can you update the CHANGELOG? In the "Added" section

Reply:
@carmocca OK.